auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -309,6 +311,10 @@ public SignInIntentBuilder createSignInIntentBuilder() {
         return new SignInIntentBuilder();
     }
 
+    public void notifyOnMergeFailedListeners(String prevUid) {
```

Comment:
I'm pretty sure this is not okay, but I don't know how to get rid of it. How would I serialize the listener into `FlowParameters`?